site/src/pages/CreateWorkspacePage/CreateWorkspacePageExperimental.tsx

```diff
@@ -202,30 +199,33 @@ const CreateWorkspacePageExperimental: FC = () => {
 						autoCreateWorkspaceMutation.error
 					}
 					resetMutation={createWorkspaceMutation.reset}
-					template={templateQuery.data!}
+					template={templateQuery.data ?? ({} as Template)}
```

Comment:
could we do an early return with a loader or an error instead? `{}` is definitely not a valid `Template` so this is just passing the buck; moving the bug somewhere else instead of being a type error